packages/patdiff/patdiff.v0.15.0/opam

```diff
@@ -18,7 +18,7 @@ depends: [
   "patience_diff"            {>= "v0.15" & < "v0.16"}
   "ppx_jane"                 {>= "v0.15" & < "v0.16"}
   "dune"                     {>= "2.0.0"}
-  "pcre"
+  "pcre" {>= "0"}
```

Comment:
```suggestion
  "pcre"
```
Why this bound? The opam repository does have only > 6 versions...

Reply:
I don't know why this was added. @kit-ty-kate  - can we revert this change to patdiff?

Reply:
Kate is away right now, let's re-add and check if the CI is still happy